.ado/jobs/e2e-test.yml

```diff
@@ -101,12 +101,62 @@ jobs:
           targetFolder: $(Build.StagingDirectory)/ReactUWPTestAppTreeDump/reports
         condition: succeededOrFailed()
 
+      - task: PowerShell@2
+        displayName: "get app info"
+        inputs:
+          targetType: "inline"
+          script: "get-appxpackage -Name ReactUWPTestApp*"
+        condition: succeededOrFailed()
+
+      - task: PowerShell@2
+        displayName: "get users"
+        inputs:
+          targetType: "inline"
+          script: "get-localuser"
+        condition: succeededOrFailed()
+
+      - task: PowerShell@2
+        displayName: "dir reactuwptestapp"
+        inputs:
+          targetType: "inline"
+          script: "gci C:\\Users | % { $_; gci $_\\AppData\\Local\\Packages\\ReactUWPTestApp_cezq6h4ygq1hw -Recurse -ErrorAction SilentlyContinue }"
+        condition: succeededOrFailed()
+
+      - task: CmdLine@2
+        displayName: print env
+        inputs:
+          script: set 
+          workingDirectory:  $(LocalAppData)\Packages
+        condition: succeededOrFailed()
+
+      - task: CmdLine@2
+        displayName: dir users
+        inputs:
+          script: dir C:\Users 
+          workingDirectory:  $(LocalAppData)\Packages
+        condition: succeededOrFailed()
+
+      - task: CmdLine@2
+        displayName: output json path
+        inputs:
+          script: type $(UserProfile)\Documents\outputJsonPath.txt
+          workingDirectory:  $(LocalAppData)\Packages
+        condition: succeededOrFailed()
+
+      - task: CmdLine@2
+        displayName: type with full path
+        inputs:
+          script: type C:\Users\VssAdministrator\AppData\Local\Packages\ReactUWPTestApp_cezq6h4ygq1hw\LocalState\ControlStyleRegularBorder.json
+          workingDirectory:  $(LocalAppData)\Packages
+        condition: succeededOrFailed()
+
+
       - task: CopyFiles@2
         displayName: Copy tree dump output files
         inputs:
-          sourceFolder: $(LocalAppData)\Packages\ReactUWPTestApp_kc2bncckyf4ap\LocalState
+          sourceFolder: $(LocalAppData)\Packages\ReactUWPTestApp_cezq6h4ygq1hw\LocalState
```

Comment:
Should we create a pipeline variable/parameter for this value?

Reply:
makes sense - I'm still waiting to hear back from virtual environments team on what I'm seeing where the app sees the files being created, but after the app exits, the files are not there: filed https://github.com/actions/virtual-environments/issues/2207